src/sentry/conf/server.py

```diff
@@ -852,6 +852,8 @@ def create_partitioned_queues(name):
     "organizations:integrations-vsts-limited-scopes": False,
     # Enable PagerDuty metric alerts
     "organizations:integrations-pagerduty-metric-alerts": False,
+    # Use Sentry Apps with Metric Alerts
+    "organizations:integrations-metric-alerts-support": False,
```

Comment:
@mgaeta I think we should have `sentry-apps` here instead of `integrations` so it's more clear what this feature flag does.